test/Trace.tests.ts

```diff
@@ -71,7 +60,6 @@ describe('Trace', () => {
 				it('combines the traces', () => {
```

Comment:
This test no longer adds anything (combining is no longer visible if the traces have no records). Let's add a few records to improve it.